documentation/user-interface/run-configurations-dialog.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: docs
+title: Run Configurations Dialog
+permalink: /documentation/user-interface/run-configurations-dialog.html
+---
+# Run Configurations Dialog
+
+The **Run Configurations** dialog allows you to create, modify and delete [run configurations](../working-with-testcafe-studio/running-tests.md#run-configurations).
+The dialog opens when you click the ![Settings button](../../images/working-with-testcafe-studio/settings-icon.png) button next to the **Run Configuration** drop-down menu on the main toolbar.
+
+![Run Configurations Dialog](../../images/working-with-testcafe-studio/run-configurations-dialog.png)
+
+The dialog's left panel displays the list of available run configurations. When you select a configuration in the list, the dialog's right section displays the configuration settings organized into several categories.
+
+**General**
```

Comment:
Maybe we should make this a header? It will be more convenient to post links to particular options.

Reply:
Yes, I'll do this.